source/features/ci-link.tsx

```diff
@@ -22,7 +22,7 @@ const getPRIcon = onetime(async () => {
 		await fetchDom(buildRepoURL('pull', getConversationNumber()!, 'commits'));
 
 	// TS bug does not allow us to directly return this 🤷‍♂️
-	const icon = select('.js-commits-list-item:last-of-type .commit-build-statuses', base);
+	const icon = select.last('.js-commits-list-item:last-of-type .commit-build-statuses', base);
```

Comment:
Doesn't :last-of-type already do this? If not, is it needed anymore?

Reply:
Also does the new fetchDom type let us use `return select` now?

Reply:
I think its still needed. (Can't check right now)

>The commits can span over a few days, we need the last one on the list

We need the last group and the last one on the list.

Reply:
> Also does the new fetchDom type let us use `return select` now?

No it does not